csstype ColorAdjust is deprecated and causes error on build

### Description

I expect to be able to compile my project using Vite. The project did not compile but instead I received the error message " Namespace '"/home/frank/git/chakra-ui-ColorAdjust-bug/node_modules/csstype/index".Property' has no exported member 'ColorAdjust'."

### Link to Reproduction

https://github.com/honungsburk/chakra-ui-coloradjust-bug

### Steps to reproduce

1. Set up the repository I have linked
2. `npm install`
3. `npm run build`

should produce the errors:
```bash

node_modules/@chakra-ui/menu/dist/declarations/src/use-menu.d.ts:480:50 - error TS2694: Namespace '"/home/frank/git/chakra-ui-ColorAdjust-bug/node_modules/csstype/index".Property' has no exported member 'ColorAdjust'.

480         colorAdjust?: import("csstype").Property.ColorAdjust | undefined;
                                                     ~~~~~~~~~~~

node_modules/@chakra-ui/menu/dist/declarations/src/use-menu.d.ts:986:61 - error TS2694: Namespace '"/home/frank/git/chakra-ui-ColorAdjust-bug/node_modules/csstype/index".Property' has no exported member 'ColorAdjust'.

986         WebkitPrintColorAdjust?: import("csstype").Property.ColorAdjust | undefined;
                                                                ~~~~~~~~~~~
```

### Chakra UI Version

1.8.6

### Browser

_No response_

### Operating System

- [ ] macOS
- [ ] Windows
- [X] Linux

### Additional Information

Seems like if "ColorAdjust" is replaced with "PrintColorAdjust" everything compiles. I can update my node_modules/ manually until there is a real fix.

I'm having this same issue. I am able to fix this bug manually in my local machine but what about Vercel?
 
```
node_modules/@chakra-ui/menu/dist/declarations/src/use-menu.d.ts(480,50): error TS2694: Namespace '"/vercel/path0/node_modules/csstype/index".Property' has no exported member 'ColorAdjust'.
--
21:16:23.083 | node_modules/@chakra-ui/menu/dist/declarations/src/use-menu.d.ts(986,61): error TS2694: Namespace '"/vercel/path0/node_modules/csstype/index".Property' has no exported member 'ColorAdjust'.
21:16:23.120 | error Command failed with exit code 2.
```

> I'm having this same issue. I am able to fix this bug manually in my local machine but what about Vercel?
> 
> ```
> node_modules/@chakra-ui/menu/dist/declarations/src/use-menu.d.ts(480,50): error TS2694: Namespace '"/vercel/path0/node_modules/csstype/index".Property' has no exported member 'ColorAdjust'.
> --
> 21:16:23.083 | node_modules/@chakra-ui/menu/dist/declarations/src/use-menu.d.ts(986,61): error TS2694: Namespace '"/vercel/path0/node_modules/csstype/index".Property' has no exported member 'ColorAdjust'.
> 21:16:23.120 | error Command failed with exit code 2.
> ```

On Vercel works fine

> > I'm having this same issue. I am able to fix this bug manually in my local machine but what about Vercel?
> > ```
> > node_modules/@chakra-ui/menu/dist/declarations/src/use-menu.d.ts(480,50): error TS2694: Namespace '"/vercel/path0/node_modules/csstype/index".Property' has no exported member 'ColorAdjust'.
> > --
> > 21:16:23.083 | node_modules/@chakra-ui/menu/dist/declarations/src/use-menu.d.ts(986,61): error TS2694: Namespace '"/vercel/path0/node_modules/csstype/index".Property' has no exported member 'ColorAdjust'.
> > 21:16:23.120 | error Command failed with exit code 2.
> > ```
> 
> On Vercel works fine

But then why I'm having this issue 🤔. Which version are you using?

I am having

> ### Description
> I expect to be able to compile my project using Vite. The project did not compile but instead I received the error message " Namespace '"/home/frank/git/chakra-ui-ColorAdjust-bug/node_modules/csstype/index".Property' has no exported member 'ColorAdjust'."
> 
> ### Link to Reproduction
> https://github.com/honungsburk/chakra-ui-coloradjust-bug
> 
> ### Steps to reproduce
> 1. Set up the repository I have linked
> 2. `npm install`
> 3. `npm run build`
> 
> should produce the errors:
> 
> ```shell
> node_modules/@chakra-ui/menu/dist/declarations/src/use-menu.d.ts:480:50 - error TS2694: Namespace '"/home/frank/git/chakra-ui-ColorAdjust-bug/node_modules/csstype/index".Property' has no exported member 'ColorAdjust'.
> 
> 480         colorAdjust?: import("csstype").Property.ColorAdjust | undefined;
>                                                      ~~~~~~~~~~~
> 
> node_modules/@chakra-ui/menu/dist/declarations/src/use-menu.d.ts:986:61 - error TS2694: Namespace '"/home/frank/git/chakra-ui-ColorAdjust-bug/node_modules/csstype/index".Property' has no exported member 'ColorAdjust'.
> 
> 986         WebkitPrintColorAdjust?: import("csstype").Property.ColorAdjust | undefined;
>                                                                 ~~~~~~~~~~~
> ```
> 
> ### Chakra UI Version
> 1.8.6
> 
> ### Browser
> _No response_
> 
> ### Operating System
> * [x]  macOS
> * [ ]  Windows
> * [ ]  Linux
> 
> ### Additional Information
> Seems like if "ColorAdjust" is replaced with "PrintColorAdjust" everything compiles. I can update my node_modules/ manually until there is a real fix.

I am having the same issue

Hello! Im also facing this issue in a recent Vite project ive created
```
> tsc && vite build

node_modules/@chakra-ui/menu/dist/declarations/src/use-menu.d.ts:480:50 - error TS2694: Namespace '"/Users/aaron/personal_code/react-sessions/react-state-management-demo/node_modules/csstype/index".Property' has no exported member 'ColorAdjust'.

480         colorAdjust?: import("csstype").Property.ColorAdjust | undefined;
                                                     ~~~~~~~~~~~

node_modules/@chakra-ui/menu/dist/declarations/src/use-menu.d.ts:986:61 - error TS2694: Namespace '"/Users/aaron/personal_code/react-sessions/react-state-management-demo/node_modules/csstype/index".Property' has no exported member 'ColorAdjust'.

986         WebkitPrintColorAdjust?: import("csstype").Property.ColorAdjust | undefined;
                                                                ~~~~~~~~~~~


Found 2 errors in the same file, starting at: node_modules/@chakra-ui/menu/dist/declarations/src/use-menu.d.ts:480
```

Ive attempted adding the resolution for `csstype` as suggested above but it had no affect on the issue for me.
Is there any more info available on this one? Cheers!

Hello! Im also facing this issue in a recent Vite project ive created
```
> tsc && vite build

node_modules/@chakra-ui/menu/dist/declarations/src/use-menu.d.ts:480:50 - error TS2694: Namespace '"/<OMITTED>/node_modules/csstype/index".Property' has no exported member 'ColorAdjust'.

480         colorAdjust?: import("csstype").Property.ColorAdjust | undefined;
                                                     ~~~~~~~~~~~

node_modules/@chakra-ui/menu/dist/declarations/src/use-menu.d.ts:986:61 - error TS2694: Namespace '"/<OMITTED>/node_modules/csstype/index".Property' has no exported member 'ColorAdjust'.

986         WebkitPrintColorAdjust?: import("csstype").Property.ColorAdjust | undefined;
                                                                ~~~~~~~~~~~


Found 2 errors in the same file, starting at: node_modules/@chakra-ui/menu/dist/declarations/src/use-menu.d.ts:480
```

Ive attempted adding the resolution for `csstype` as suggested above but it had no affect on the issue for me.
Is there any more info available on this one? Cheers!

> Hello! Im also facing this issue in a recent Vite project ive created
> 
> ```
> > tsc && vite build
> 
> node_modules/@chakra-ui/menu/dist/declarations/src/use-menu.d.ts:480:50 - error TS2694: Namespace '"/<OMITTED>/node_modules/csstype/index".Property' has no exported member 'ColorAdjust'.
> 
> 480         colorAdjust?: import("csstype").Property.ColorAdjust | undefined;
>                                                      ~~~~~~~~~~~
> 
> node_modules/@chakra-ui/menu/dist/declarations/src/use-menu.d.ts:986:61 - error TS2694: Namespace '"/<OMITTED>/node_modules/csstype/index".Property' has no exported member 'ColorAdjust'.
> 
> 986         WebkitPrintColorAdjust?: import("csstype").Property.ColorAdjust | undefined;
>                                                                 ~~~~~~~~~~~
> 
> 
> Found 2 errors in the same file, starting at: node_modules/@chakra-ui/menu/dist/declarations/src/use-menu.d.ts:480
> ```
> 
> Ive attempted adding the resolution for `csstype` as suggested above but it had no affect on the issue for me. Is there any more info available on this one? Cheers!

same issues

> Hello! Im also facing this issue in a recent Vite project ive created
> 
> ```
> > tsc && vite build
> 
> node_modules/@chakra-ui/menu/dist/declarations/src/use-menu.d.ts:480:50 - error TS2694: Namespace '"/<OMITTED>/node_modules/csstype/index".Property' has no exported member 'ColorAdjust'.
> 
> 480         colorAdjust?: import("csstype").Property.ColorAdjust | undefined;
>                                                      ~~~~~~~~~~~
> 
> node_modules/@chakra-ui/menu/dist/declarations/src/use-menu.d.ts:986:61 - error TS2694: Namespace '"/<OMITTED>/node_modules/csstype/index".Property' has no exported member 'ColorAdjust'.
> 
> 986         WebkitPrintColorAdjust?: import("csstype").Property.ColorAdjust | undefined;
>                                                                 ~~~~~~~~~~~
> 
> 
> Found 2 errors in the same file, starting at: node_modules/@chakra-ui/menu/dist/declarations/src/use-menu.d.ts:480
> ```
> 
> Ive attempted adding the resolution for `csstype` as suggested above but it had no affect on the issue for me. Is there any more info available on this one? Cheers!

I checked the package.json of csstype under node_modules folder, and it's version is listed at 3.0.11. And yet it still shows this error. I'd appreciate if someone can help me about this issue, thanks!

@seancaleb It should be fixed at 3.0.10, not 3.0.11.

> > Hello! Im also facing this issue in a recent Vite project ive created
> > ```
> > > tsc && vite build
> > 
> > node_modules/@chakra-ui/menu/dist/declarations/src/use-menu.d.ts:480:50 - error TS2694: Namespace '"/<OMITTED>/node_modules/csstype/index".Property' has no exported member 'ColorAdjust'.
> > 
> > 480         colorAdjust?: import("csstype").Property.ColorAdjust | undefined;
> >                                                      ~~~~~~~~~~~
> > 
> > node_modules/@chakra-ui/menu/dist/declarations/src/use-menu.d.ts:986:61 - error TS2694: Namespace '"/<OMITTED>/node_modules/csstype/index".Property' has no exported member 'ColorAdjust'.
> > 
> > 986         WebkitPrintColorAdjust?: import("csstype").Property.ColorAdjust | undefined;
> >                                                                 ~~~~~~~~~~~
> > 
> > 
> > Found 2 errors in the same file, starting at: node_modules/@chakra-ui/menu/dist/declarations/src/use-menu.d.ts:480
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Ive attempted adding the resolution for `csstype` as suggested above but it had no affect on the issue for me. Is there any more info available on this one? Cheers!
> 
> I checked the package.json of csstype under node_modules folder, and it's version is listed at 3.0.11. And yet it still shows this error. I'd appreciate if someone can help me about this issue, thanks!

--- a/.changeset/tender-pugs-sniff.md
+++ b/.changeset/tender-pugs-sniff.md
@@ -1,0 +1,5 @@
+---
+"@chakra-ui/styled-system": patch
+---
+
+Update `csstype` package to fix typings error for `colorAdjust` property

--- a/package.json
+++ b/package.json
@@ -117,7 +117,7 @@
     "chart.js": "^3.6.0",
     "commitizen": "4.2.4",
     "cross-env": "7.0.3",
-    "csstype": "3.0.9",
+    "csstype": "^3.0.11",
     "cz-conventional-changelog": "3.3.0",
     "discord.js": "^12.5.3",
     "dotenv-cli": "4.1.0",

--- a/packages/styled-system/package.json
+++ b/packages/styled-system/package.json
@@ -37,7 +37,7 @@
   },
   "dependencies": {
     "@chakra-ui/utils": "2.0.0-next.1",
-    "csstype": "3.0.9"
+    "csstype": "^3.0.11"
   },
   "sideEffects": false
 }

